tasks/build/download_node_builds.js

```diff
@@ -63,22 +130,18 @@ module.exports = function (grunt) {
       });
     }());
 
-    activeDownloads.push(platform);
+    const success = await checkShaSum(platform);
+    if (success !== true) throw (new Error(`${platform.name} download failed`));
 
-    var bytes = parseInt(resp.headers['content-length'], 10) || 'unknown number of';
-    var mb = ((bytes / 1024) / 1024).toFixed(2);
-    grunt.log.ok(`downloading ${platform.name} - ${mb} mb`);
+    grunt.log.ok(`${platform.name} download complete`);
   };
 
-  grunt.registerTask('_build:downloadNodeBuilds:start', function () {
-    map(platforms, start).nodeify(this.async());
+  grunt.registerTask('_build:downloadShaSums', function () {
```

Comment:
I don't think downloadShaSums should be a separate grunt task, since downloadNodeBuilds has a hard dependency on it.